README.md

```diff
@@ -75,157 +79,205 @@ W3C Verifiable Credentials specification defines two kinds of objects – Verifi
 
 This specification defines the JWT parameters to pass VPs and VCs signed as JWTs or using Linked Data Proofs.
 
-This specifications also introduces a new artifact VP Token and VC Token to provide VPs and VCs to RPs:
-
-### JWT parameters extention
+This specifications also introduces a new artifact VP Token and VC Token to provide VPs and VCs to RPs.
 
-This specification introduces the following JWT parameters:
+All representations share the same container format.
 
-- vc_jwt:  A claim whose value is a W3C Verifiable Credential object using the JWT representation, which is a JSON string.  The claim’s value may also be an array of W3C Verifiable Credential objects using the JWT representation if the use case calls for multiple JWT VCs.
+## Container Format
 
-- vp_jwt:  A claim whose value is a W3C Verifiable Presentation object using the JWT representation, which is a JSON string.  The claim’s value may also be an array of W3C Verifiable Presentation objects using the JWT representation if the use case calls for multiple JWT VPs.
+### Verifiable Credential
 
-- vc_ldp:  A claim whose value is a W3C Verifiable Credential object using the JSON-LD representation, which is a JSON object.  The claim’s value may also be an array of W3C Verifiable Credential objects using the JSON-LD representation if the use case calls for multiple JSON-LD VCs.
+A verifiable credential container is an array of objects, each of them containing the following fields:
 
-- vp_ldp:  A claim whose value is a W3C Verifiable Presentation object using the JSON-LD representation, which is a JSON object.  The claim’s value may also be an array of W3C Verifiable Presentation objects using the JSON-LD representation if the use case calls for multiple JSON-LD VPs.
+`format`: REQUIRED A JSON string denoting the proof format the credential was returned in. This specification introduces the values `w3cvc-jsonld` and `w3cvc-jwt` to denote credentials in JSON-LD and JWT format, respectively.  
```

Comment:
Technically, it does not mean that VP in JSON format is signed as a JWT, it can also be signed as LD-Proof. For VP in JSON-LD, I think the only meaningful option is signing using LD-Proofs. 
Probably we need both fields `credential_format` - JSON and JSON-LD and `proof_format` - JWT or LD-Proof

Reply:
@awoie @AdamJLemmon what do you think?

Reply:
I would use the format designators from DIF PE [here](https://identity.foundation/presentation-exchange/#claim-format-designations) which are: jwt_vc, jwt_vp, ldp_vc, ldp_vp. It also defines jwt and ldp but I don't know how they are used.

I agree with @Sakurann that JSON-LD is not a proof type and I agree that the VC can be interpreted as plain JSON since the [VC spec says](https://www.w3.org/TR/vc-data-model/#contexts):
> Though this specification requires that a @context property be present, it is not required that the value of the @context property be processed using JSON-LD. This is to support processing using plain JSON libraries, such as those that might be used when the verifiable credential is encoded as a JWT. All libraries or processors MUST ensure that the order of the values in the @context property is what is expected for the specific application. Libraries or processors that support JSON-LD can process the @context property using full JSON-LD processing as expected.

However, since in practice, each VC should have a valid `@context` property and valid URIs, therefore being valid JSON-LD, I'm not sure if this is really required. If you feel it makes a difference, I'm ok with adding support for that case.

Reply:
changed identifiers to follow DIF PE definition

Reply:
I believe the intent here was that `format` was referring to the proof format not the serialization to JSON or JSON-LD beforehand.

I tend to think just the single format property for proof format using the values @awoie has suggested from PE should suffice. I'd prefer to not complicate with both properties, but if the case can be made that this would simplify RP processing than I am open?